src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.2.0'
+__version__ = '1.2.1'
```

Comment:
We're adding a new field `uid`, so this should be a minor rather than patch bump.  I know it feels weird given the magnitude of the change...

Reply:
Updated to 1.3.0